packages/remix-react/package.json

```diff
@@ -23,6 +23,8 @@
     "@remix-run/server-runtime": "1.17.0",
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^13.3.0",
+    "@types/react": "^18.0.15",
+    "@types/react-dom": "^18.0.6",
```

Comment:
Do we actually need the react-dom types?

Reply:
No, we only had `react` before